lib/datadog/tracing/sampling/span/sampler.rb

```diff
@@ -27,14 +27,15 @@ module Span
         # that has been accepted by trace-level sampling rules: all spans from such
         # trace are guaranteed to reach the Datadog App.
         class Sampler
+          attr_reader :rules
```

Comment:
Is there a reason to make this public? I did not find other using this, or is this only a shortcut for test?

Reply:
Short cut for testing.

Reply:
The class itself is not part of the public API either.